diablo/models/sign_up.py

```diff
@@ -0,0 +1,106 @@
+"""
+Copyright ©2020. The Regents of the University of California (Regents). All Rights Reserved.
+
+Permission to use, copy, modify, and distribute this software and its documentation
+for educational, research, and not-for-profit purposes, without fee and without a
+signed licensing agreement, is hereby granted, provided that the above copyright
+notice, this paragraph and the following two paragraphs appear in all copies,
+modifications, and distributions.
+
+Contact The Office of Technology Licensing, UC Berkeley, 2150 Shattuck Avenue,
+Suite 510, Berkeley, CA 94720-1620, (510) 643-7201, otl@berkeley.edu,
+http://ipira.berkeley.edu/industry-info for commercial licensing opportunities.
+
+IN NO EVENT SHALL REGENTS BE LIABLE TO ANY PARTY FOR DIRECT, INDIRECT, SPECIAL,
+INCIDENTAL, OR CONSEQUENTIAL DAMAGES, INCLUDING LOST PROFITS, ARISING OUT OF
+THE USE OF THIS SOFTWARE AND ITS DOCUMENTATION, EVEN IF REGENTS HAS BEEN ADVISED
+OF THE POSSIBILITY OF SUCH DAMAGE.
+
+REGENTS SPECIFICALLY DISCLAIMS ANY WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE. THE
+SOFTWARE AND ACCOMPANYING DOCUMENTATION, IF ANY, PROVIDED HEREUNDER IS PROVIDED
+"AS IS". REGENTS HAS NO OBLIGATION TO PROVIDE MAINTENANCE, SUPPORT, UPDATES,
+ENHANCEMENTS, OR MODIFICATIONS.
+"""
+
+from diablo import db
+from diablo.models.base import Base
+from sqlalchemy.dialects.postgresql import ARRAY, ENUM
+
+
+publish_type = ENUM(
+    'canvas',
+    'kaltura_media_gallery',
+    name='publish_types',
+    create_type=False,
+)
+
+recording_type = ENUM(
+    'presentation_audio',
+    'presenter_audio'
+    'presenter_presentation_audio',
+    name='recording_types',
+    create_type=False,
+)
+
+
+class SignUp(Base):
+    __tablename__ = 'sign_ups'
+
+    term_id = db.Column(db.Integer, nullable=False, primary_key=True)
+    section_id = db.Column(db.Integer, nullable=False, primary_key=True)
+    admin_approval_uid = db.Column(db.String)
+    instructor_approval_uids = db.Column(ARRAY(db.String))
+    publish_type = db.Column(publish_type, nullable=False)
+    recording_type = db.Column(recording_type, nullable=False)
+    requires_multiple_approvals = db.Column(db.Boolean, nullable=False)
+
+    def __init__(
+            self,
+            term_id,
+            section_id,
+            admin_approval_uid,
+            instructor_approval_uids,
+            publish_type_,
+            recording_type_,
+            requires_multiple_approvals,
```

Comment:
Notes:
* This table is likely to get another column: `recordings_scheduled` (boolean). 
* Time will tell if `requires_multiple_approvals` is the right way to manage workflow. A sign-up can sit in limbo if (a) instructor #1 says "yes" to course capture and (b) soon after, instructor #2 is dropped from the course. A course-capture admin will need to intervene.